PhotoPlusFrontend/PhotoPlus/src/app/models/order/order.ts

```diff
@@ -0,0 +1,24 @@
+import { Product } from '../product/product';
+
+export class Order {
+  userCode: number;
+  orderStatus: string;
+  paymentMethod: string;
+  price: number;
+  date: string;
+  addressCode: number;
+  orderItems: [{
+    orderCode: string;
+    productCode: number;
+    quantity: number;
+  }]
+  address: {
```

Comment:
you should create separate model for OrderItem and Address entity

Reply:
For address is already created in my branch. Mock it somehow for now.